docs/Makefile

```
@@ -2,13 +2,22 @@
 
 IMAGE = grafana/docs-base:latest
 
-docs:
-	docker pull ${IMAGE}
+docs-generate-frontend:
+	yarn install --pure-lockfile --no-progress
+	yarn run packages:build
+	yarn run packages:docsExtract
+	yarn run packages:docsToMarkdown
+	yarn run packages:clean
+
+docs-no-generate-frontend:
 	docker run -v $(shell pwd)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it $(IMAGE) /bin/bash -c 'make server'
 
-docs-no-pull:
+docs: docs-generate-frontend
+	docker pull ${IMAGE}
```

Can we use the `$()` form to be consistent with the rest of the file?
```suggestion
	docker pull $(IMAGE)
```

---

Sure, I have not changed this but why not :)

---

OK, I see it's inconsistent already, but mostly parens are used.